scripts/src/primazactl/primazamain/parser.py

```diff
@@ -0,0 +1,112 @@
+import os
+import argparse
+from pathlib import Path
+from primazactl.errors import AtLeastOneError
+from primazactl.types import \
+    existing_file, kubernetes_name, semvertag_or_latest
+from .primazamain import PrimazaMain
+from primazactl.utils.kubeconfig import from_env
+
+
+def add_group(subparsers, parents=[]):
+    main_group = subparsers.add_parser(
+        name="main",
+        help="Operations on main Primaza cluster",
+        parents=parents)
+    main_subparsers = main_group.add_subparsers()
+
+    add_subparser_install(main_subparsers, parents=parents)
```

Comment:
In the refactor primazactl has changed from:
- do this to this (e.g.: install main)
to
- to this do this (e.g.: main install)

most commands kubectl, helm, minikube, brew, etc are of the form "do this to this", and a a result the first level help you get a list of all the things you can do.

Also with other commands you get the flags at the second level of help, for example `kubectl  apply --help`
With the new refactor only the third level of help shows flags.

Any reason for the change?

Reply:
the reason is that we have two main context we can work on: main and worker cluster.
I agree on the following the `kubectl` approach, but I think it would be better to have it scoped to `main` cluster or `worker` cluster. In this way, the user can rely on `primazactl main -h` to get the list of all commands that are applicable on a main cluster and `primazactl worker -h` to get the list of commands that applies to workers.

Reply:
OK, so we are kind of making two commands:  primazamainctl and primazaworkerctl but wrapping it into one. That seems OK.